src/redux/services/api.ts

```diff
@@ -143,14 +152,23 @@ export const api = createApi({
       invalidatesTags: (result, error, arg) => [{ type: 'ProjectExtraData', id: arg.uuid }, { type: 'OfferMessage' }],
     }),
 
+    // GET: Fetch all reservations for a specific apartment
+    getApartmentReservations: builder.query<ApartmentReservationWithCustomer[], string>({
+      query: (apartmentId) => `apartments/${apartmentId}/reservations`,
```

Comment:
```suggestion
      query: (apartmentId) => `apartments/${apartmentId}/reservations/`,
```
The URL actually ends with a slash in the API, so that should be included to avoid an unnecessary redirect.

Reply:
This is now fixed